File: BackUpForABook/UIControls/Graph_framework/Graph/Graph/TreeRingMap/TRGridLine.m
//
//  TRGridLine.m
//  ATreeRingMap
//
//  Created by hui wang on 12/20/15.
//  Copyright (c) 2015 AnnieLyticx. All rights reserved.
//

#import "TRGridLine.h"

@implementation TRGridLine

-(id)init
{
    self = [super init];
    
    _style = GridLine_Solid;
    _size = 2;
    _color = [UIColor whiteColor];
    
    return self;
}

@end

File: BackUpForABook/UIControls/TRClasses_swift/TRGridLine.swift
//
//  TRGridLine.swift
//  TreeRingMap_Swift
//
//  Created by dingf on 17/3/16.
//  Copyright © 2017年 MH.dingf. All rights reserved.
//

import UIKit

class TRGridLine: NSObject {
    var color: UIColor?
    var size: Float?
    var style: GridLine?

    override init() {
        super.init()
        style = .solid
        size = 2.0
        color = UIColor.white
    }
}
